lib/dicom_net/association.ex

```diff
@@ -158,9 +158,24 @@ defmodule DicomNet.Association do
     # TODO this is not always c-store
 
     ds = Dicom.BinaryFormat.from_binary(data, ts_options)
-    msg = {:dicom, %{operation: :cstore, dataset: ds}}
-    send(event_listener, msg)
-    response_ds = handle_cstore(command, ds)
+    commandField = command[256].values 
+
+    response_ds = case commandField do
+        [1] -> IO.inspect("C-Store")
```

Comment:
Would stay with hex literals in this code, as this is how the DICOM standard describes it as well. Easier to map then.

Reply:
Yes, I used [1] and [256] because I don't know if I could use, for example 0x0001 directly, or [0x0001]. Which is the recommended way?

Reply:
What I mean is the following:

On line 161 you access a named field in the command dataset, which you best do with:
```
command_field = DataSet.fetch!(:CommandField)
```

Then:
```
case command_field do
  0x01 -> ...
  0x20 -> ...
```

This way I can compare the code directly to the DICOM standard (in this case [here](https://dicom.nema.org/dicom/2013/output/chtml/part07/sect_9.3.html#table_9.3-3)), and see that it's correct without converting the number in my head.

Reply:
Thanks!, that's what I needed.